pysteps/nowcasts/steps.py

```diff
@@ -463,72 +477,78 @@ def f(R, i):
             noise_std_coeffs = np.ones(n_cascade_levels)
 
         if noise_stddev_adj is not None:
-            print("noise std. dev. coeffs:   %s" % str(noise_std_coeffs))
+            print(f"noise std. dev. coeffs:   {str(noise_std_coeffs)}")
+    else:
+        pp = None
+        noise_std_coeffs = None
 
     # compute the cascade decompositions of the input precipitation fields
-    R_d = []
+    precip_d = []
```

Comment:
If changed in the previous scripts, based on my comments, then change it also here. I have to say that `precip_d` also works for me, so whatever we collectively prefer. 

The same holds for `precip_c` below.